charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -26,6 +26,11 @@ spec:
   serverVersion: {{ .Values.cassandra.version | quote }}
   dockerImageRunsAsCassandra: true
   serverImage: {{ include "k8ssandra.cassandraImage" . }}
```

Comment:
All of the following lines have the new `k8ssandra-common.flattenedImage` helper. Why is `serverImage` not using this?

Reply:
The `k8ssandra.cassandraImage` uses `k8ssandra-common.flattenedImage`.  `cassandraImage` has the logic for the version/image mapping as well.

Reply:
I see where the miss is here. For some reason, I parsed this as just dumping a value not calling the helper (which then calls flattenedImage). Resolving.